generate_templates.py

```diff
@@ -207,24 +208,35 @@ def main():
     interfacemap = json.loads(DEVICEMAP_PATH.read_text())
 
     for vendor, skus in interfacemap["interfaceMap"].items():
+        models = set()
         for sku, devicemap in skus.items():
             actual_devicemap = resolve_alias(devicemap, interfacemap)
             if not actual_devicemap:
                 continue
-            template = generate_template(vendor, sku, actual_devicemap)
-            write_template(vendor, sku, template)
 
-def generate_template(vendor, sku, devicemap):
+            model = devicemap.get('displayModel') or sku
+
+            # make sure models are unique per vendor
+            if model in models:
+                print("Vendor {vendor} has duplicate model {model}")
+                sys.exit(1)
```

Comment:
Should we actually exit here? I think we should just not generate a template for the duplicate. I could see the `displayModel` being used for something like the madrids where there are a whole bunch of them just slightly different.

Reply:
I disagree I think. We don't want certain devices to not have a unique template. Or else, someone could use a template for another device that isn't appropriate

Reply:
And discussed offline, we can change this behavior in the future if we run into a roadblock